demo/src/search/search-page.js

```diff
@@ -134,41 +134,33 @@ export class SearchPage extends LitElement {
     return this.#abortController.signal;
   }
 
-  #openPlayer(event) {
-    const button = event.target.closest('button');
-
-    if (!('urn' in button.dataset)) return;
-
-    openPlayerModal({ src: button.dataset.urn, type: 'srgssr/urn' });
+  #toQueryParams(r) {
+    return new URLSearchParams({ ...router.queryParams, src: r.urn, type: 'srgssr/urn' }).toString();
   }
 
   #renderButton(r) {
     const date = new Intl.DateTimeFormat('fr-CH').format(new Date(r.date));
     const duration = Pillarbox.formatTime(r.duration / 1000);
 
     return html`
-      <button class="content-btn" data-urn="${r.urn}" title="${r.title}">
-        <span class="content-btn-title">${r.title}</span>
-        <div class="content-btn-metadata-container">
-          <i
-            class="material-icons-outlined">${r.mediaType === 'VIDEO' ? 'movie' : 'audiotrack'}</i>
-          <span class="content-btn-info">&nbsp;| ${date} | ${duration}</span>
-        </div>
-      </button>
+      <content-link title="${r.title}"
+                   href="search?${this.#toQueryParams(r)}">
+          <i class="material-icons-outlined">${r.mediaType === 'VIDEO' ? 'movie' : 'audiotrack'}</i>
+          <span>&nbsp;| ${date} | ${duration}</span>
```

Comment:
The icon and date/duration isn't rendered without something like
```suggestion
<div slot="description">
          <i class="material-icons-outlined">${r.mediaType === 'VIDEO' ? 'movie' : 'audiotrack'}</i>
          <span>&nbsp;| ${date} | ${duration}</span>
</div>
```